src/test/java/com/icthh/xm/ms/balance/service/LepContextCastIntTest.java

```diff
@@ -79,7 +79,7 @@ public void testLepContextCast() {
         String prefix = "/config/tenants/RESINTTEST/balance/lep/service/";
         String key = prefix + "GetBalanceChangesByTypeAndDate$$around.groovy";
         String body = "import org.springframework.data.domain.PageImpl;\n" +
-                "import com.icthh.xm.ms.balance.config.LepContext;\nLepContext context = lepContext\n" +
+                "import com.icthh.xm.ms.balance.config.LepContext;\nLepContext context = lepContext as LepContext\n" +
```

Comment:
Don't do this! You must keep LepContext actual, this test check it! Remove farse cast and add fields to lepContext

Reply:
added in lepContext field LepServiceFactory lepServices